buildkit/orafce.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: v1
+name: orafce
+version: "4.7.0"
+homepage: https://github.com/orafce/orafce
+repository: https://github.com/orafce/orafce
+source: https://github.com/orafce/orafce/archive/refs/tags/VERSION_4_7_0.tar.gz
+description: The "orafce" project implements in Postgres some of the functions from the Oracle database that are missing (or behaving differently).Those functions were verified on Oracle 10g, and the module is useful for production work.
+licence: BSD
+keywords:
+  - oracle
+  - functions
+  - 10g
+arch:
+  - amd64
+  - arm64
+maintainers:
+  - name: James Lovern
+    email: james@lovern.io
+buildDependencies:
+  - libicu-dev
+  - libpq-dev
+  - bison
+build:
+  main:
+    - name: Build orafce
+      run: |
+        make USE_PGXS=1
+        make install
```

```suggestion
        DESTDIR=${DESTDIR} make install
```

honestly not sure if this is needed, we may have cargo-culted our way to having this in most buildkits. If the extension works as-is, no need to add this.

---

tested with setting DESTDIR and works so leaving it in for worst-case.